pkg/fleet/internal/exec/installer_exec.go

```diff
@@ -42,15 +42,13 @@ type installerCmd struct {
 }
 
 func (i *InstallerExec) newInstallerCmd(ctx context.Context, command string, args ...string) *installerCmd {
-	env := i.env.ToEnv()
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 Warning:** ‏You can't really remove this, as technically `i.env` can be [populated from a config file](https://github.com/DataDog/datadog-agent/blob/db5c3f8609cd02e7327b96c35f4ef5797911dd9c/pkg/fleet/env/env.go#L61-L69) (when running in the daemon for instance).

Reply:
nice catch, fixed